webrtc.html

```diff
@@ -523,39 +524,46 @@
           </li>
         </ol>
 
-        <p>Once the RTCPeerConnection object has been initialized, for every
-        call to <code>createOffer</code>, <code>setLocalDescription</code>,
-        <code>createAnswer</code>, <code>setRemoteDescription</code>,
-        and <code>addIceCandidate</code>,
-        execute the following steps:</p>
+        <p>An <code><a>RTCPeerConnection</a></code> object has an
+        <dfn>operations queue</dfn>, [[<a>operations</a>]], which ensures that
+        only one asynchronous operation in the queue is executed concurrently. If
+        subsequent calls are made while the returned promise of a previous call
+        is still not settled, they are added to the queue and executed when all
```

Comment:
It's from [states and fates](https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md) at least.